### PR TITLE
FixingUserRoles

### PR DESCRIPTION
**Pull request by CatholicCardinal**
-create enum UserRoles
-fixing SeedExtention
-move UserRoles adn SeedExtention to Bll layer
-rename TokenService to AuthenticationService
-delete register
-delete unused models UI

### CODE REVIEW

File: BLL/SeedExtension.cs [hunk 1]
```diff
@@ -8,19 +9,19 @@
...
 
-namespace DAL
+namespace BLL
 {
     public static class StartupSeedExtension
     {
         public static async Task SeedRoles(RoleManager<IdentityRole> roleManager)
         {
-            string[] roleNames = { "admin", "recruiter", "manager", "interviewer", "mentor", "supermentor" };
+            UserRoles[] roleNames = { UserRoles.admin, UserRoles.recruiter, UserRoles.manager, UserRoles.interviewer, UserRoles.mentor, UserRoles.supermentor };
```
Comment (VasiliNesterenko) [inline BLL/SeedExtension.cs:18]: Please use enumeration of ENUM object

File: UI/Attribute/OptionalAuthorizeAttribute.cs [hunk 2]
```diff
@@ -12,7 +12,7 @@
 using Microsoft.Extensions.DependencyInjection;
 using BLL.Interfaces;
 
-namespace UI
+namespace UI.Attribute
 {
     public class BlacklistAuthorizeAttribute : AuthorizeAttribute, IAuthorizationFilter
```
Comment (VasiliNesterenko) [inline UI/Attribute/OptionalAuthorizeAttribute.cs:17]: Please correct filename: OptionalAuthorizeAttribute.cs  vs BlacklistAuthorizeAttribute

**Review by VasiliNesterenko — CHANGES_REQUESTED** [review]
Please review my comments

File: UI/Attribute/BlacklistAuthorizeAttribute.cs [hunk 3]
```diff
@@ -12,7 +12,7 @@
 using Microsoft.Extensions.DependencyInjection;
 using BLL.Interfaces;
 
-namespace UI
+namespace UI.Attribute
```
Comment (DShcharbakou) [inline UI/Attribute/BlacklistAuthorizeAttribute.cs:15]: .Attribute**s**

File: UI/Controllers/HomeController.cs [hunk 4]
```diff
@@ -9,22 +8,24 @@
...
 using Microsoft.Extensions.Caching.Memory;
+using UI.Attribute;
 
 namespace UI.Controllers
 {
     [ApiController]
     [Route("[controller]")]
     public class HomeController : Controller
     {
+        UserRoles claim = UserRoles.admin;
```
Comment (DShcharbakou) [inline UI/Controllers/HomeController.cs:19]: For which purpose did you set up this one line?

File: UI/Controllers/HomeController.cs [hunk 5]
```diff
@@ -9,22 +8,24 @@
...
         [BlacklistAuthorize]
-        [Authorize(Roles = UserRoles.Admin)]
+        [Authorize()]
```
Comment (DShcharbakou) [inline UI/Controllers/HomeController.cs:21]: If you need using role use enum as a parameter for this attribute

**Review by DShcharbakou — APPROVED** [review]

**Review by DShcharbakou — COMMENTED** [review]
See comments above

**Review by DShcharbakou — CHANGES_REQUESTED** [review]
See comments above

**Review by VasiliNesterenko — APPROVED** [review]

**Review by AnnKirienko — APPROVED** [review]

**Review by DShcharbakou — APPROVED** [review]

**Pull request merged**